packages/vscode-extension/package.json

```diff
@@ -327,12 +327,12 @@
       },
       {
         "command": "fx-extension.viewEnvironment",
-        "title": "Teams: View environment profile",
+        "title": "View environment",
```

Comment:
why not keep the prefix `Teams: ` like above command?

Reply:
This will be shown on the UI. There are no ```Teams:``` on the tree view by design.